torch/distributed/rpc/api.py

```diff
@@ -543,6 +551,14 @@ def rpc_async(to, func, args=None, kwargs=None):
         args (tuple): the argument tuple for the ``func`` invocation.
         kwargs (dict): is a dictionary of keyword arguments for the ``func``
                        invocation.
+        timeout (Optional[datetime.timedelta]): timeout to use for this RPC. If the RPC
+                                                does not complete in this amount of
+                                                time, an exception indicating it has
+                                                timed out will be raised. If not
+                                                provided, the default value set during
+                                                initialization or with _set_rpc_timeout
```

Comment:
I saw `_set_rpc_timeout` is removed from `api.py`, should we still mention it here?

Reply:
I didn't remove it in this PR, it still exists in `rpc/init.cpp`. As you mentioned, I think that we will still need this to control timeouts for internal messages.